src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -166,11 +168,14 @@ export const ImportAccounts = ({ chainId, onClose }: ImportAccountsProps) => {
     () => wallet instanceof MetaMaskShapeShiftMultiChainHDWallet,
     [wallet],
   )
+  const isMetaMaskButNotSnap = isMetaMaskMultichainWallet && isSnapInstalled === null
   const chainNamespaceDisplayName = asset?.networkName ?? ''
   const [autoFetching, setAutoFetching] = useState(true)
   const [queryEnabled, setQueryEnabled] = useState(false)
   const [isSubmitting, setIsSubmitting] = useState(false)
   const [toggledAccountIds, setToggledAccountIds] = useState<Set<AccountId>>(new Set())
+  const isLoadMoreDisabledForThisChain =
+    !!isSnapInstalled && (isEvmChainId(chainId) || isCosmosSdkChainId(chainId))
```

Comment:
we might as well check the wallet for `supportsBip44Accounts()` as well and can use supports multi account venacular
```suggestion
  const supportsMultiAccount = useMemo(() => {
    if (!wallet?.supportsBip44Accounts()) return false
    if (wallet instanceof MetaMaskShapeShiftMultiChainHDWallet) {
      return Boolean(isSnapInstalled) && !isEvmChainId(chainId) && isCosmosChainId(chainId)
    }
    return true
  }, [chainId, isSnapInstalled, wallet])
```

Reply:
Alternatively for MM (i.e l.179-181 in the suggestion above), can use `canAddMetaMaskAccount`, using `lastPage.accountNumber + 1` as the account number to check against.
That would ensure we don't duplicate the Cosmos SDK/EVM chains checks logic in too many places.
Note we should do preliminary MM checks if using that because that method will throw if we're using any adapter other than MM Multichain:

https://github.com/shapeshift/web/blob/ad2301dcf04bff83e7d5b6de419ade2910cb2315/src/pages/Accounts/AddAccountModal.tsx#L65-L79

Reply:
I thought about that as well and that would make sense. Was thinking magic number, but now seeing `lastPage.accountNumber` so I like that idea. Good call!

Reply:
Took your suggestions in accounts, pushed the fix!